development.md

```diff
@@ -1,3 +1,5 @@
+Contributions via PR to this repo are welcome. Please for the repo, work on your feature or change using the guide below, and then make a PR against this repo. The team will review the PR and work with you to get it merged. Thanks!
```

Comment:
Should we change the beginning of this 2nd sentence to "Please work on your feature..."?